python/ray/data/_internal/stats.py

```diff
@@ -988,6 +989,14 @@ def get_total_wall_time(self) -> float:
             for ss in self.operators_stats
         )
 
+    def get_total_time_all_blocks(self) -> float:
+        parent_total_times = [p.get_total_time_all_blocks() for p in self.parents]
+        parent_time_total = sum(parent_total_times) if parent_total_times else 0
+        return parent_time_total + sum(
+            ss.wall_time.get("sum", 0) if ss.wall_time else 0
+            for ss in self.operators_stats
+        )
```

Comment:
this is really getting the total wall time across all operators, not across blocks right? would a more appropriate name for the method be `get_total_time_all_operators`?

Reply:
I think that name makes more sense, but because we use `ss.wall_time.get("sum", 0)` that is a sum across all blocks within the operator.